server/src/main/kotlin/suwayomi/tachidesk/manga/controller/UpdateController.kt

```diff
@@ -22,4 +38,54 @@ object UpdateController {
             }
         )
     }
+
+    fun categoryUpdate(ctx: Context) {
+        val categoryId = Optional.ofNullable(ctx.formParam("category")).map { it.toIntOrNull() }.orElse(null)
```

Comment:
```suggestion
        val categoryId = ctx.formParam("category")?.toIntOrNull()
```
Kotlins null system is basically Optional